pkg/assets/handlers.go

```diff
@@ -188,16 +188,12 @@ func addExtensionStylesheets(content []byte, extensionStylesheets []string) []by
 
 var versionTemplate = template.Must(template.New("webConsoleVersion").Parse(`
 window.OPENSHIFT_VERSION = {
-  openshift: "{{ .OpenShiftVersion | js}}",
```

Comment:
you no longer need to expose this?

Reply:
Not needed anymore. We had to switch the request to the client anyway to fix a bug: 

https://github.com/openshift/origin-web-console/pull/2734